src/app.js

```
@@ -1 +1,56 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+function copyFile() {
+  const source = process.argv[2];
+  const destination = process.argv[3];
+
+  if (!source || !destination) {
+    console.error('Missing arguments');
+
+    return;
+  }
+
+  if (!fs.existsSync(source)) {
+    console.error('File does not exist');
+
+    return;
+  }
+
+  const sourceStats = fs.statSync(source);
+
+  if (sourceStats.isDirectory()) {
+    console.error('Source is a directory');
+
+    return;
+  }
+
+  if (fs.existsSync(destination)) {
+    const destinationStats = fs.statSync(destination);
+
+    if (destinationStats.isDirectory()) {
+      console.error('Destination is a directory');
+
+      return;
+    }
+  }
+
+  if (source === destination) {
+    return;
```

Issue: When the source and destination are the same, the function silently returns without any feedback. If the task requires user feedback for all error cases, you should print an error message here to inform the user that source and destination must be different.